manifests/i/Igoravl/TfsCmdlets/2.0.1/Igoravl.TfsCmdlets.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Igoravl.TfsCmdlets
+PackageVersion: 2.0.1.2474
+PackageLocale: en-US
+Publisher: Igor Abade V. Leite
+PublisherUrl: https://github.com/igoravl/TfsCmdlets
+PublisherSupportUrl: https://github.com/igoravl/TfsCmdlets/issues
+#PrivacyUrl: 
+Author: Igor Abade V. Leite
+PackageName: TfsCmdlets
+PackageUrl: https://tfscmdlets.dev/
+License: MIT
+LicenseUrl: https://github.com/igoravl/TfsCmdlets/blob/main/LICENSE.md
+Copyright: Copyright (c) Igor Abade V. Leite
+#CopyrightUrl: 
+ShortDescription: PowerShell Cmdlets for Azure DevOps and Team Foundation Server
+Description: TfsCmdlets is a PowerShell module which provides many commands to simplify automated interaction with Azure DevOps (Server & Services) and Team Foundation Server.
+Moniker: tfscmdlets
+Tags:
+- TfsCmdlets
+- TFS
+- VSTS
+- PowerShell
+- Azure
+- AzureDevOps
+- DevOps
+- ALM
+- TeamFoundationServer
```

Also, best practice is to use all lower case tags. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
Tags:
- tfscmdlets
- tfs
- vsts
- powershell
- azure
- azuredevops
- devops
- alm
- teamfoundationserver
```